cmd/flexctl/backups.go

```diff
@@ -224,3 +227,126 @@ func listBackups(cmd *cobra.Command) error {
 func backupsEnabled() bool {
 	return os.Getenv("S3_ARCHIVE_CONFIG") != ""
 }
+
+func newBackupConfigCmd() *cobra.Command {
+	var configCmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manage backup configuration",
+	}
+
+	configCmd.AddCommand(newConfigShowCmd())
+	configCmd.AddCommand(newConfigUpdateCmd())
+
+	return configCmd
+}
+
+type configShowResult struct {
+	Result flypg.BarmanSettings `json:"result"`
+}
+
+func newConfigShowCmd() *cobra.Command {
+	var configShowCmd = &cobra.Command{
+		Use:   "show",
+		Short: "Show current configuration",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			resp, err := http.Get("http://localhost:5500/commands/admin/settings/view/barman")
+			if err != nil {
+				return err
+			}
+
+			var rv configShowResult
+			err = json.NewDecoder(resp.Body).Decode(&rv)
+			if err != nil {
+				return err
+			}
+
+			fmt.Printf("  ArchiveTimeout = %s\n", rv.Result.ArchiveTimeout)
+			fmt.Printf("  RecoveryWindow = %s\n", rv.Result.RecoveryWindow)
+			fmt.Printf("  FullBackupFrequency = %s\n", rv.Result.FullBackupFrequency)
+			fmt.Printf("  MinimumRedundancy = %s\n", rv.Result.MinimumRedundancy)
+
+			return nil
+		},
+	}
+
+	return configShowCmd
+}
+
+type successfulUpdateResult struct {
+	Message         string `json:"message,omitempty"`
+	RestartRequired bool   `json:"restart_required"`
+}
+
+type configUpdateResult struct {
+	Result successfulUpdateResult `json:"result,omitempty"`
+	Error  string                 `json:"error,omitempty"`
+}
+
+func newConfigUpdateCmd() *cobra.Command {
+	var configUpdateCmd = &cobra.Command{
+		Use:   "update",
+		Short: "Update configuration",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			update := flypg.BarmanSettings{
```

Comment:
It's generally more idiomatic to access flags using the approach: 

archiveTimeout, err :=  cmd.Flag().GetString("archive-timeout")
if err != nil {
   ...
}